docs/using-concourse/fly-cli.any

```diff
@@ -415,16 +415,21 @@ flag. Once you've learned what the commands do, you may want to consult
   }
 
   To check from a particular version, including the given version, append the
-  \code{--from} flag like so:
+  \code{--from} flag. For example, to check for version '11.2' and above on
+  the \italic{bosh-io-release} resource, use the following:
 
   \codeblock{bash}{
   $ fly -t example check-resource --resource my-pipeline/my-resource \\
-      --from ref:abcdef
+      --from version:11.2
```

Comment:
I prefer using `ref:abcdef` as the example; `version:` looks more like a keyword than `ref` to me. We should call out that it's for the `git` resource though.